daemon/volumes.go

```diff
@@ -120,7 +121,8 @@ type volumeStore struct {
 
 type volumeCounter struct {
 	volume.Volume
-	count int
+	count      int
+	containers []string
```

Comment:
what if we use a map to store the containers? we can make deleting a container id O(1) by doing something like this:

```go
containers map[string]bool
...
containers[refName] = true
...
delete(containers, refName)
```